app/src/main/java/fr/neamar/kiss/loader/LoadShortcutsPojos.java

```diff
@@ -71,10 +93,30 @@ protected List<ShortcutPojo> doInBackground(Void... params) {
         return pojos;
     }
 
+    @RequiresApi(35)
+    private boolean shouldAddShortcut(UserManager manager, LauncherUserInfo info,
```

Comment:
Method to check if shortcuts should be hidden is already there, it's `ShortcutUtil.isShortcutVisible` some lines above.
Can you add the additional check there, as there is already some more logic depending on visibility of shortcuts.